src/JsonRpc/JsonRpcServer.php

```diff
@@ -289,11 +254,8 @@ protected function requestFromEnvelope(Envelope $envelope): Request
             throw new Exception\InvalidJsonRpcRequest();
         }
 
-        $payload = json_decode($envelope->getBody(), true);
-
-        if (0 !== json_last_error()) {
-            throw new Exception\JsonParseError();
-        }
+        // @todo introduce json util ?
+        $payload = \json_decode($envelope->getBody(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
What is this todo about?

Reply:
something similar to https://github.com/prooph/event-store/blob/master/src/Util/Json.php

Reply:
Why not?